desktop/src/select/select/select.component.scss

```
@@ -0,0 +1,19 @@
+// beside the option text and in the trigger, never replacing either.
+.select__badge {
+  display: inline-block;
+  margin-left: 6px;
+  vertical-align: middle;
+  font-size: 9.5px;
+  font-weight: 700;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  border-radius: 5px;
+  padding: 2px 6px;
+  // Same WCAG reasoning as the report config panel's badge: 9.5px/700 is small
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)